example/README.md

```diff
@@ -0,0 +1,87 @@
+# Overview
+This directory contains a simple example/demo that we can provide to any user who wants to learn how concord-bft works and how they may use it. 
```

Comment:
Eventually a bit more elaboration can be added here

Reply:
I have added some more details about the replica part. This will be an ongoing task and I will update the documentation when I will add client and other execution engines.